### PR TITLE
Feature/average rating

### PR DESCRIPTION
**Pull request by ChrisRPeterson**
Added function getAverageRatingFromId in App.js. Created a useEffect to call the function when productId changes and stores the result in state.

### CODE REVIEW

**Comment by ChrisRPeterson** [comment]
It looks like it has a lot of changes that were already merged into main? Sorry, I'll look into why that is, but the relevant changes are in App.js

File: client/components/App.jsx [hunk 1]
```diff
@@ -37,8 +40,44 @@ const theme = createMuiTheme({
...
+
+  if (numberOfRatings === 0) {
+    return 5;
+  }
+  return sumOfRatings / numberOfRatings;
+};
+
 const App = () => {
   const [productId] = useState(24156);
+  const [averageRating, setAverageRating] = useState(5);
```
[the remark this answers is not in the document]
Comment (ChrisRPeterson) [reply client/components/App.jsx:70]: I thought we wouldn't want our products to look like they have 0 stars, but 5 doesn't feel quite right either 😂

[the remark this answers is not in the document]
Comment (ChrisRPeterson) [reply client/components/App.jsx]: @julia-thea I changed it so it defaults to 0 now

[the remark this answers is not in the document]
Comment (julia-thea) [reply client/components/App.jsx]: Lolz I see your point 😄

**Review by julia-thea — APPROVED** [review]
Thanks so much for implementing this, good stuff.

**Pull request merged**